pom.xml

```diff
@@ -71,7 +71,7 @@
 		<dependency>
 			<groupId>org.apache.tomcat</groupId>
 			<artifactId>tomcat-catalina</artifactId>
-			<version>8.0.39</version>
```

Comment:
There were some more changes necessary to support tomcat 8.5: https://github.com/chexagon/redis-session-manager/commit/aa4fed0ea4761756da3c55e5c51cdf71357d80a6

I'd recommend leaving the tomcat dependency alone (or upgrading to 8.0.47) and letting 8.5 support be handled in a different branch / PR.